### PR TITLE
update python api readme

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by VincyZhang — APPROVED** [review]

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 1]
```diff
@@ -81,6 +81,30 @@ outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
 
 ```
 
+Augument description of generate:
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:84]:
```suggestion
Argument description of generate function:
```

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 2]
```diff
@@ -81,6 +81,30 @@ outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
...
+| Augument             | Description                                                              |
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:85]:
```suggestion
| Argument             | Description                                                              |
```

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 3]
```diff
@@ -81,6 +81,30 @@ outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
...
+| --------------       | -----------------------------------------------------------------------  |
+| inputs               | input ids after tokenizer                                                |
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:87]: better to indicate the datatype type like
```suggestion
| inputs             | uint32_t | input ids after tokenizer                                                |
```

File: intel_extension_for_transformers/llm/runtime/graph/README.md [hunk 4]
```diff
@@ -81,6 +81,30 @@ outputs = model.generate(inputs, streamer=streamer, max_new_tokens=300)
...
+| seed                 | NG seed (default: -1, use random seed for < 0)                           |
+| threads              | number of threads to use during computation (default: 8)                 |
+| repetition_penalty   | penalize repeat sequence of tokens (default: 1.1, 1.0 = disabled)        |
+| num_beams            | number of beams for beam_search (default: 1)                             |
+| do_sample            | Whether or not to use sampling ; use greedy decoding otherwise. (default: False) |
+| top_k                | top-k sampling (default: 40, 0 = disabled)                               |
+| top_p                | top-p sampling (default: 0.95, 1.0 = disabled)                           |
+| temperature          | temperature (default: 0.8)                                               |
+| min_new_tokens       | The minimum numbers of tokens to generate, ignoring the number of tokens in the prompt. |
+| length_penalty       | Exponential penalty to the length that is used with beam-based generation. |
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/README.md:103]:
```suggestion
| length_penalty      | float | Exponential penalty to the length that is used with beam-based generation.(between 0 and 1) |
```

**Review by airMeng — COMMENTED** [review]
I think arguments explanation should be aligned with help info? can you align the help info here too?
https://github.com/intel/intel-extension-for-transformers/blob/66238a524b7968b43cf1117ef420e6e0f6782d16/intel_extension_for_transformers/llm/runtime/graph/scripts/run.py#L36

File: intel_extension_for_transformers/llm/runtime/graph/README.md (hunk 1 above)
Comment (zhenwei-intel) [reply intel_extension_for_transformers/llm/runtime/graph/README.md]: fix, thx

File: intel_extension_for_transformers/llm/runtime/graph/README.md (hunk 3 above)
Comment (zhenwei-intel) [reply intel_extension_for_transformers/llm/runtime/graph/README.md:87]: uint32_t is not used by python...

Comment (airMeng) [reply intel_extension_for_transformers/llm/runtime/graph/README.md:87]: then ```int``` should be enough

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by airMeng — APPROVED** [review]

**Review by PenghuiCheng — APPROVED** [review]

**Pull request merged**